Blocks/Persistence/file.h

```diff
@@ -129,48 +146,49 @@ class FilePersister {
 
  private:
   struct FilePersisterImpl final {
-    const std::string filename;
-    std::ofstream appender;
-    std::fstream head_rewriter;
-
-    // `offset.size() == end.next_index`, and `offset[i]` is the offset in bytes where the line for index `i` begins.
-    std::mutex& mutex_ref;  // Guards `offset`, `head_offset` and `timestamp`.
-    std::vector<std::streampos> offset;
-    std::streamoff head_offset;
-    std::vector<std::chrono::microseconds> timestamp;
-
-    // Just `std::atomic<end_t> end;` won't work in g++ until 5.1, ref.
+    const std::string filename_;
+    std::ofstream file_appender_;
+    std::fstream head_rewriter_;
+
+    // `record_offset_.size() == end.next_index`,
+    // and `record_offset_[i]` is the record_offset_ in bytes where the line for index `i` begins.
+    std::mutex& publish_mutex_ref_;  // Guards `record_offset_`, `head_offset_` and `record_timestamp_`.
+    std::vector<std::streampos> record_offset_;
+    std::streampos head_offset_;
+    std::vector<std::chrono::microseconds> record_timestamp_;
+
+    // Just `std::atomic<end_t> end_;` won't work in g++ until 5.1, ref.
     // http://stackoverflow.com/questions/29824570/segfault-in-stdatomic-load/29824840#29824840
-    // std::atomic<end_t> end;
-    current::atomic_that_works<end_t> end;
+    // std::atomic<end_t> end_;
```

Comment:
I suggest we stay conservative here. This part is to be retired soon regardless -- as we'd be moving to C++14, I assume.

Thoughts?